### ISSUE TITLE
Electron apps in system tray do not respond to left click

### ISSUE THREAD

**Issue by assen-totin** [mate-desktop/mate-panel#1049]
#### Expected behaviour
Single-click or double-click of a system tray icon would always open the corresponding app. If the app was already open, but minimsed (i.e. has a button on the task switcher in the panel), it would still bring it up.

#### Actual behaviour
Since one of the latest Electron releases, Electron apps no longer follow this paradigm. Left click on their icon, both single and double, does nothing. Example apps are latest Skype and Slack versions. 

#### Steps to reproduce the behaviour
1. Install Slack or Skype on a MATE desktop.
2. Start the app.
3. Use the "X" button to minimise the app to system tray.
4. Click the system tray icon of the app to restore it.

#### MATE general version
1.22

#### Package version
1.22.1-1

#### Linux Distribution
Fedora 29, Fedora 30

**Comment by raveit65** [comment]
Are electron developers informed for using skype with Mate?

**Comment by assen-totin** [comment]
No, but I would not blame Electron - as the issue started the moment it
switched from GTK to Qt for the GUI toolit. So the issue is likely with Qt
apps going to our system tray and whether they get/respond to left click at
all.

**Comment by raveit65** [comment]
Dropbox or keepassxc na-tray applets are using Qt too and they are working fine, so, i am still wondering what make's the different?
And how should a developer fix the issue without to know what is change in electron code base?
In other words, where should we begin?

**Comment by assen-totin** [comment]
Hm, don't know what Dropbox uses, but on my PC it does not respond to a
left click in the system tray (I'm not sure if it ever did, though).

If I had a clue where to start from, I would have started :) Maybe I should
write an Electron app an test it locally? No idea, really.

**Comment by raveit65** [comment]
> Hm, don't know what Dropbox uses, but on my PC it does not respond to a
left click in the system tray (I'm not sure if it ever did, though).

That's a different when SNI suport is enabled. Try disable SNI support and left-click will work.

**Comment by assen-totin** [comment]
There are some suggestions that the issue is related to the use of
libappindicator3, which we get from GTK3. And some reports that without it,
Electron apps work fine. I know that Electron will use libappindicator if
it is available. Also, the issue started with Electron v3 and later, v2 is
reported to work properly.
